events/producer/events.go

```diff
@@ -274,10 +270,11 @@ func AddEventType(eventType pb.EventType) error {
 }
 
 func registerHandler(ie *pb.Interest, h *handler) error {
-	producerLogger.Debugf("registerHandler %s", ie.EventType)
+	//producerLogger.Debugf("registerHandler %s", ie.EventType)
```

Comment:
Yea, I commented it out intentionally as it was getting displayed during the unit tests. I can add back if you would like or just remove.